pkg/clients/gitlab.go

```diff
@@ -191,3 +192,17 @@ func IsIntEqualToIntPtr(ip *int, i int) bool {
 	}
 	return true
 }
+
+// IsStringEqualToIntPtr compares an *int with string
+func IsStringEqualToIntPtr(ip *string, i int) bool {
```

Comment:
The description says `*int` but the parameter is `int`.

```suggestion
// IsStringEqualToIntPtr compares an int with string
func IsStringEqualToIntPtr(ip *string, i int) bool {
```